### ISSUE TITLE
[Discussion] Improve the handlign of OpticalChannel in metadata 

### ISSUE THREAD

**Issue by h-mayorquin** [catalystneuro/neuroconv#605]
To discuss what to do about this. 

To start the discussion two current issues:

1) The optical channels now can only through adding an imaging plane:

https://github.com/catalystneuro/neuroconv/blob/567213d4b255a62d6b83e34e7a94929cf16c10b2/src/neuroconv/tools/roiextractors/roiextractors.py#L211-L213

That is, the only way to add them is to pass a dictionary with the metadata of the optical channel in the metadata of an imaging plane. Maybe we should change that?

2) As @alessandratrapani points out here https://github.com/catalystneuro/clandinin-lab-to-nwb/issues/13 the metadata is not even accepted in the current schema. That is, you can't pass an `OpticalChannel` key at the top of `Ophys` schema.

This should probably be discussed in the context of:
https://github.com/catalystneuro/neuroconv/issues/269

**Comment by h-mayorquin** [comment]
In the link that you are sharing the OpticalChannel metadata is passed through the metadata of `ImagingPlane` as I am claiming here.

>  That is, you can't pass an OpticalChannel key at the top of Ophys schema.

**Comment by CodyCBakerPhD** [comment]
Right, but that's intended so long as our neurodata types are the code NWB version

**Comment by h-mayorquin** [comment]
OK, so nothing has changed. Adding OpticalChannels any other way was never supported. 

If this will change only with `ndx-microscopy` then we can close this and re-open at that point. As you assigned me on this on I thought that you expected differently:

https://github.com/catalystneuro/clandinin-lab-to-nwb/issues/13

**Comment by CodyCBakerPhD** [comment]
Oh, the assignments were just for dashboard tracking, which got derailed a while ago and I was fixing some of it yesterday lol
